integration-tests/suites/k8s/config_reload.go

```diff
@@ -0,0 +1,165 @@
+package k8s
+
+import (
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"github.com/stackrox/collector/integration-tests/pkg/log"
+
+	coreV1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	EXT_IP_ENABLE = `
+networking:
+  externalIps:
+    enable: true
+`
+
+	EXT_IP_DISABLE = `
+networking:
+  externalIps:
+    enable: false
```

Comment:
I also tested invalid input in my PR, but maybe that can be done here later in another PR.

Reply:
Also I had a function that accepted a bool, converted it to a string, and added it to the config string. I think that seems a bit better than this.

Reply:
I will consider doing these on follow up PRs, right now the tests are minimal and focused on correct functionality, not error handling that has already been tested in unit tests.

Reply:
> Also I had a function that accepted a bool, converted it to a string, and added it to the config string. I think that seems a bit better than this.

There's probably a way to define the full structure and serialize it to yaml that would be easier than this described approach, for now, the current approach is good enough for the implemented tests IMO.

Reply:
Okay